gem/backends.py

```diff
@@ -62,14 +67,29 @@ def filter_users_by_claims(self, claims):
         """
         uuid = claims["sub"]
         try:
-            kwargs = {USERNAME_FIELD: uuid}
+            kwargs = {'profile__auth_service_uuid': uuid}
```

Comment:
can we put an except here for if the profile does not exist, and things for what do if that happens?